oauth2/00-setup.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+SCRIPT_DIR="$(dirname "${BASH_SOURCE[0]}")"
+
+. "$SCRIPT_DIR/../.common.sh"
+. "$SCRIPT_DIR/.support.sh"
+
+# Setup configuration
+
+# Delete the realm if it already exists
+exec_ssoadm $TEST_INSTANCE_ID delete-realm \
+  --adminid amadmin \
+  --password-file pwd.txt \
+  --realm "$TEST_REALM" > /dev/null || :
+
+# Create new realm
+exec_ssoadm $TEST_INSTANCE_ID create-realm \
+  --adminid amadmin \
+  --password-file pwd.txt \
+  --realm "$TEST_REALM" \
+  > /dev/null
+
+# Create OAuth2 Provider Service
+#
+# Special settings different from defaults:
+#   * Allow clients to skip consent: enabled
+#   * Issue refresh tokens: enabled
+#
+# Provided XML file with service configuration was obtained via "export-svc-cfg" ssoadm command
+#
+OAUTH_SVC_CFG="oauth2-svc-cfg.xml"
+
+# Copy service configuration file to container and replace placeholders
+docker cp "$SCRIPT_DIR/cfg/$OAUTH_SVC_CFG" "wrenam-test$TEST_INSTANCE_ID:/opt/ssoadm/auth/$OAUTH_SVC_CFG"
+exec_am $TEST_INSTANCE_ID bash -c "sed -i -e 's|{{REALM}}|$TEST_REALM|' /opt/ssoadm/auth/$OAUTH_SVC_CFG" > /dev/null
```

Comment:
Pls drop the XML and create svc via add-svc-realm and properties.

Reply:
Done.